proposals/evaluation/uc-3-inheritance.md

```diff
@@ -199,18 +236,19 @@ However, if WAC's use of `acl:default` were to be relaxed as described in [issue
 
 See also: https://github.com/solid/authorization-panel/pull/216#discussion_r665230245
 
+The problem here is that we need to make a copy of `</weekly-status/2021-04-28/.acl>` in the new acl, which means that changes to the root acl do not get propagated, and the more changes there are the more places will need to be edited to make any changes.
```

Comment:
```suggestion
```

This is exactly the opposite of what is described here.

With `acl:default` relaxed, you don't need any copies, you get all the authorizations in the effective ACR applied.

Reply:
Thanks for spotting all those little mistakes. 
This paragraph I think was meant to be placed in the previous section. I need a bigger screen to get a better overview.

Reply:
Ok, I moved that and made other fixes.
I think these changes should be getting better, as I get into it, after having been away for a week or more.